### ISSUE TITLE
Ability To Define Message Title?

### ISSUE THREAD

**Issue by cjastennett** [meg768/homebridge-pushover#14]
Hi, 
is there some way to define the Message title, rather than it just being "HomeBridge" for every message that gets sent? If not, is it possible to add that functionality?
Thanks - otherwise, great plugin!
cheers
Chris

**Comment by meg768** [comment]
Never really thought about that but it is must possible...

**Comment by meg768** [comment]
Wait, du you mean a title per message? One could add a "title" entry in the config file...

**Comment by cjastennett** [comment]
Yes, configuring the plug-in to allow a title attribute in the configuration file to be picked up and sent to pushover would be great. I think that should be a relatively straightforward modification looking at the code.

**Comment by Galapgs** [comment]
I've created a fork and a pull request for title and Pushover priority

**Comment by meg768** [comment]
Yes. New version on npm.js. Thanks to **Galapgs** (and of course **cjastennett**) .

**Comment by Galapgs** [comment]
Thanks for the fast adjustments!

**Comment by meg768** [comment]
> Thanks for the fast adjustments!

Well, fast adjustments but not really tested. Let me now if it doesn't work.
